apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/pac/R4ClaimSamhsaMatcherTransformerTest.java

```diff
@@ -374,28 +418,60 @@ public static Stream<Arguments> mcsTest() {
     return Stream.of(
         arguments(
             "SAMHSA ICD 9 Diagnosis code",
-            List.of("0:" + NON_SAMHSA_CODE, "1:" + ICD_9_DX_SAMHSA_CODE, "0:" + NON_SAMHSA_CODE),
+            List.of("0:" + NON_SAMHSA_CODE, "9:" + ICD_9_DX_SAMHSA_CODE, "0:" + NON_SAMHSA_CODE),
```

Comment:
Pretty sure setting `1` instead of `9` for the ICD type here was an accident. It just wouldn't be caught because an unknown system is treated as SAMHSA.